validate-charts/action.yml

```diff
@@ -9,11 +9,13 @@ inputs:
     description: 'Arguments to be appended to lint and template commands such as "--set global.env=dev"'
     required: true
     default: ''
-
 runs:
-  using: 'docker'
-  image: '../helm.dockerfile'
-  args:
-    - validate
-    - ${{ inputs.chart-path }}
-    - ${{ inputs.extra-args }}
\ No newline at end of file
+  using: "composite"
+  steps:
+    - name: validate helm chart
+      shell: bash
+      run: |
+        helm version
```

Comment:
If, we use ubuntu as github runner image, we will have helm 3.12.0, would that be fine in all the repos?
- https://github.com/actions/runner-images/blob/main/images/linux/Ubuntu2204-Readme.md

Reply:
So, my understanding is that, we will be fine with latest version of helm across repos.

And, rest of the actions like:
- helm repo add will be added into individual repo workflow whoever wants it like - https://github.com/Traceableai/traceable-platform/pull/207/files#diff-404033996c34f9971e0c112d8538b9063618fc85059a44df7d4db448974888ca